src/views/errors/ErrorDetails.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <section class="section">
+    <div class="columns is-mobile is-multiline">
+      <div class="column is-min-width-fit-content">
+        <div class="big-title-style page-header--title">Detalhe do Erro</div>
+      </div>
+      <div class="column is-min-width-fit-content">
+        <div class="buttons has-addons is-pulled-right">
+          <button class="button is-primary" @click="archiveItem">
+            <span class="icon"><i class="fa fa-archive"></i></span>
+            <span>Arquivar</span>
+          </button>
+          <button class="button is-danger" @click="deleteItem">
+            <span class="icon"><i class="fa fa-trash-alt"></i></span>
+            <span>Excluir</span>
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="box">
+      <div class="columns is-mobile is-multiline is-vcentered">
+        <div class="column">
+          <div class="medium-title-style">
+            {{ error.name }} no {{ error.origin }}
+          </div>
+          <div class="subtitle date">{{ error.last_date | formatDate }}</div>
+          <div class="margin-top">
+            <div class="small-title-style">
+              Título
+            </div>
+            {{ error.title }}
+          </div>
+          <div class="margin-top">
+            <div class="small-title-style">
+              Detalhes
+            </div>
+            <div v-for="(detail, index) in error.details" :key="index">
+              {{ detail }}
+            </div>
+          </div>
+        </div>
+        <div class="column is-min-width-fit-content column-message">
+          <article class="message">
+            <div class="message-body" :class="classMessage">
+              <div class="tag is-medium" :class="classMessage">
+                {{ error.level }}
+              </div>
+              <div class="label-style">Eventos</div>
+              <div class="small-title-style">{{ error.qty }}</div>
+              <div class="label-style">Usuário</div>
+              <div class="small-title-style">{{ error.collected_by }}</div>
+            </div>
+          </article>
+        </div>
+      </div>
+    </div>
+  </section>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+
+export default {
+  props: {
+    id: String,
+  },
+  data() {
+    return {
+      error: {},
+    };
+  },
+  computed: {
+    ...mapGetters,
+    classMessage() {
+      // eslint-disable-next-line no-nested-ternary
```

Comment:
porque esse comentario?

Reply:
para não bugar o lint do ternário

Reply:
agora não precisou mais do comentário tirei o ternário que dava problema

Reply:
entendi, acho que dá pra alterar para isso então:
`return this.error.level || 'debug';`

ou

```
const { level } = this.error;
return level || 'debug';
```

Reply:
não precisa mais, pois antes eu pegava o que vinha e fazia um ternário para as classes do bulma... do jeito que está não precisa de ternário, pega o que vem e aplica, sempre virá um deles